### ISSUE TITLE
Start streaming songs immediately

### ISSUE THREAD

**Issue by DavidVentura** [Sonerezh/sonerezh#110]
You are currently using passthru() but you are not actually reading what you return from there; you serve a file with the return, you could at least change passthru to exec().

Is there a way to actually do something like passthru? Outputting the actual contents, of the file to avoid waiting for re-encoding

**Comment by lGuillaume124** [comment]
Yes it is, but the player become unable to calculate the progress bar because it doesn't know when the song end... We will try to fix when we will work on "on-the-fly" transcoding.

**Comment by DavidVentura** [comment]
I don't know how to use Cake, if there's any way to output "on the fly" and flush() the output, I can work on this. Could you show me how to do something like this:

```
cake-on-the-fly-echo "text";
flush();
sleep(5);
return-cake-view-whatever();
```
